client/components/FAQ.js

```diff
@@ -1,18 +1,144 @@
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
 
-export default class FAQ extends React.PureComponent {
-  render () {
-    return (
-      <div>
-        <header>
-          <FormattedMessage
-            id="components.FAQ.header"
-            description="FAQ page header text"
-            defaultMessage="FAQ"
-          />
-        </header>
-      </div>
-    );
+
+import './FAQ.scss';
+import { arrayExpression } from '@babel/types';
+
+const questions = {
+    0:"How can I complete the Census?",
+    1:"What types of questions are on the Census?",
+    2:"How long will it take to complete?",
+    3:"Will the census be available in other languages?",
+    4:"I forgot my ID. Can I still take the Census?",
+    5:"Will my census information be shared with the government?",
+    6:"How do I identify a Census official in person?",
+    7:"How can I avoid scams online?",
+    8:"How do I report suspected fraud?",
+    9:"Do I need to complete the Census?",
+    10:"Why is the census important?",
+    11:"How will the Census count homeless residents?"
+};
+
+/*
+class Question extends React.Component{
+  render(props){  
+    
+    //We can _call_ functions inside our render method, but we shouldn't
+    //define them here!
+
+    function listQuestion(props){
+      return <li>{props.questions.id}</li>
+    }
+
+    function QuestionList(props){
+      const questions=props.questions;
+      const listQuestion = questions.map((questions)=>
+      <listQuestion key={props.questions.id}
+      value={props.questions.description}/>
+      );
+      return (
+        <ul>
+          {listQuestion}
+        </ul>
+      )
+    }
   }
 }
+
+renderQuestion(){
+   return<Questions value={Questions.Component} />;
+ }
+*/
+
+class Question extends React.Component {
```

Comment:
perfect case to use a react hook here

Reply:
I would be interested in discussing whether we want to go with react hooks--I'm not 100% opposed to it, just would like to explore the rationale.

Reply:
It would only be practice to see the comparison to use a hook rather than a full class component, would mainly be an exercise for seeing hooks.

Reply:
Gotcha, that would be okay.